keyboards/mutepad/readme.md

```diff
@@ -0,0 +1,33 @@
+# MutePad
+
+![MutePad Repo](https://github.com/JoshwJB/MutePad)
+
+*A short description of the keyboard/project*
```

Comment:
Needs to be filled in, or removed

Reply:
Should be updated now. Thanks for taking a look.